src/modules/quick_input_parser_module.f90

```diff
@@ -14,18 +14,48 @@ module quick_input_parser_module
 
     contains
 
-        subroutine checkformat(i,line,keyword)
+        subroutine trimSpace(i,j,line,keyword,found)
             implicit none
-            integer, intent(out) :: i
+            integer, intent(out) :: i,j
             character(len=*), intent(in) :: line
             character(len=*), intent(in) :: keyword
+            logical, intent(out) :: found
 
-            i = index(line, trim(keyword)//'=')
-            if(i==0) then
-                call PrtErr(OUTFILEHANDLE, "Error with keyword "//trim(keyword)//" encountered.")
-                call quick_exit(OUTFILEHANDLE,1)
+            print *, "keyword is ", keyword
+    
+            !first, go to the right to the end of the keyword
+            i = index(line, trim(keyword))+len_trim(keyword)
+            print *, "line(i:i+4) is", line(i:i+4)
+        
+            !ignore all spaces left to the equal sign
+            do while(line(i:i)==' ')
```

Comment:
Would be safer to also ignore tab characters. You never know what users will do.